layouts/partials/list.html

```diff
@@ -14,13 +14,13 @@
     <article class="card mb-3">
       <div class="row g-0">
         <div class="col-md-4">
-          {{ partial "helpers/image" (dict "filename" (printf "%s%s" .Params.image "?width=400px") "alt" .Title "class" "img-fluid rounded-start" "resources" .Resources) }}
+          {{ partial "helpers/image" (dict "filename" (printf "%s%s" .Params.image "?width=400px") "alt" "" "class" "img-fluid rounded-start" "resources" .Resources) }}
         </div>
         <div class="col-md-8">
           <div class="card-body">
             <h2 class="card-title post-title"><a href="{{ .Permalink }}">{{ partial "helpers/title" . }}</a></h2>
             {{- partial "post/meta" . -}}
-            <p class="card-text post-summary">{{ partial "post/excerpt" . }}</p>
+            {{ partial "post/excerpt" . }}
```

Comment:
Please replace with a div then, so that the classes and styling could still be applied.

Reply:
@elyezer I double checked and it seems like the styles are doing nothing 😅 The classes do not exist, and actually this is the structure being currently rendered on the live site:
![image](https://user-images.githubusercontent.com/9519976/195478805-8680c6ea-ee0b-442e-bed6-374ceecaf779.png)

Reply:
Something is happening with the rendering, It was not supposed to be empty, see here https://github.com/CastalioPodcast/castaliopodcast.github.io/blob/main/layouts/partials/list.html#L23

The card-text comes from the bootstrap css framework and checking there there is no special styling and the post-summary was inherited from the site when it was using pelican. With that we can go ahead and merge this one. Later I will need to improve the list template as recently I moved all content to use markdown and the excerpt needs some fixing.